portality/view/forms.py

```diff
@@ -4,12 +4,71 @@
 FIXME: probably we need to think more about how we organise forms - maybe a module with different files for different
 features, and a bit of refactoring on the organisation of the formcontext stuff to be consistent
```

Comment:
not sure why this would be in this PR, apart from that if it was missing from af_redesign in the first place

Reply:
in master, develop, and af_redesign the article form is defined within formcontext; treating this as a regression